server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==2.3.2
```

Comment:
Czy potrzebujesz flaska w dokładnie tej wersji?

Reply:
Jeśli nie masz jakichś bardzo konkretnych wymagań co do wersji to ich nie podawaj. Sama nazwa biblioteki wystarczy

Reply:
ok